comp_sci_gender_bias/getters/embedding.py

```diff
@@ -0,0 +1,23 @@
+from comp_sci_gender_bias import PROJECT_DIR
+from comp_sci_gender_bias.utils.io import load_pickle
+import torch
+from typing import Union
+import numpy as np
+import pathlib
+
+EMBEDDING_DIR = PROJECT_DIR / "outputs/embeddings"
+
+
+def embedding(
```

Comment:
this might be better as `load_embedding`? just to help with readability about what its doing (I think I'd assume it was creating embeddings)